bin/ContributeRecipe

```diff
@@ -218,7 +218,10 @@ then
    git stash pop
    git branch -D "submit-$program-$version"
 else
+   git config --global hub.protocol https
```

Comment:
This is necessary to prevent script failing as git tries to look for public key by default for ssh-git links. Potentially harmfull for other repositories on system. Should be replaced with some version affecting only local repo by I am not sure how.

Reply:
I think this becomes local if you simply remove the `--global`?